scripts/test_commands.sh

```diff
@@ -70,4 +70,6 @@ ${swctl} --display=json db g >/dev/null 2>&1
 # Test `dependency`
 ${swctl} --display=json dependency service "test" > /dev/null 2>&1
 
-${swctl} --display=json dependency endpoint "test" > /dev/null 2>&1
\ No newline at end of file
+${swctl} --display=json dependency endpoint "test" > /dev/null 2>&1
+
+${swctl} --display=json dependency endpoint "test" > /dev/null 2>&1
```

Comment:
Duplicate? Or copy-paste problem?

Reply:
My bad. It should be {swctl} --display=json dependency instance "test" "test".